### ISSUE TITLE
Regarding the issue of including negative samples in the training set

### ISSUE THREAD

**Issue by hj950815** [MIC-DKFZ/nnUNet#1924]
Hey, you can ignore these prompts. They have been removed in nnunet v2 because they are only confusing people. The way you did it is correct: just add these images to your training set with empty segmentation masks

_Originally posted by @FabianIsensee in https://github.com/MIC-DKFZ/nnUNet/issues/1412#issuecomment-1531493888_

**Comment by hj950815** [comment]
Hi, when i change oversample_foreground_percent from 0.33 to 0.7, fewer false positives, but more false negatives.It is difficult to reduce false positives while maintaining sensitivity
Is the approach proposed here a more efficient approach? Add negative images to training set with empty segmentation masks
Can you help me?

**Comment by hj950815** [comment]
At the same time I also modified batch_size to 30, patch_size 64*56*64

**Comment by GregorKoehler** [comment]
Hi @hj950815 ,
sorry for leaving this issue unanswered for so long! Can you provide some more context for your question? What is your setting and what are you trying to achieve?

**Comment by GregorKoehler** [comment]
Closing this issue for now since it's stale for roughly a week. Please feel free to re-open the issue if you're still facing this issue! If so, please provide some more context for your setting and what you're trying to achieve!
